sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md

```diff
@@ -1,14 +1,19 @@
 # Release History
 
-## 4.0.2 (2020-03-18)
+## 4.0.3 (Unreleased)
 
 ### Fixed
 
-- Fixed concurrency issue in our challenge-based authentication policy ([#9737](https://github.com/Azure/azure-sdk-for-net/issues/9737))
 - Fixed an issue where the issuer name was always null ([#10908](https://github.com/Azure/azure-sdk-for-net/issues/10908))
 - Fixed an issue where GetIssuerAsync would throw for issuers with contact information populated ([#10905](https://github.com/Azure/azure-sdk-for-net/issues/10905))
 - Fixed an issue where some Certificate policy properties were not serialized properly ([#11669](https://github.com/azure/azure-sdk-for-net/issues/11669))
 
+## 4.0.2 (2020-03-18)
+
+### Fixed
+
+- Fixed concurrency issue in our challenge-based authentication policy ([#9737](https://github.com/Azure/azure-sdk-for-net/issues/9737))
```

Comment:
From the related issue it appears that this was merged in April. Is it part of 4.0.2?

Reply:
Yep, from memory. But I wanted to be sure, so...
```
$ git tag --contains 3a64351 | findstr Certificates
Azure.Security.KeyVault.Certificates_4.0.2
```